contractor/contractor.hpp

```diff
@@ -55,6 +55,14 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 class Contractor
 {
+  public:
```

Comment:
This is actually obsolete now. We can try to merge my PR on this first and then rebase.